vignettes/epidatr.Rmd

```diff
@@ -117,7 +117,9 @@ pub_covidcast(
 )
 ```
 
-We can fetch a subset of states by listing out the desired locations:
+Alternatively, we can fetch the full time series for a subset of states by 
+listing out the desired locations in the `geo_value` argument and using `*`
+in the `time_values` argument:
```

Comment:
I guess the idea here is to demo two new features in parallel? Seems reasonable, they're pretty clearly separate

Reply:
That's right